components/mapview/page.tsx

```
@@ -1,56 +1,43 @@
 "use client";
-import mapboxgl from "mapbox-gl";
+
+import Map, { Marker } from 'react-map-gl';
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)